scripts/asset_management/audit_sync.py

```
@@ -0,0 +1,365 @@
+    L.append(f"- Inventory rows actually referenced in docs: **{a['referenced_rows_count']}**")
+    L.append(f"- Referenced in docs but NOT in inventory: **{len(a['referenced_not_in_inventory'])}** "
+             f"({len(a['referenced_not_in_inventory_images'])} are images -> caption these)")
+    L.append(f"- Dead rows (404 / source_type=missing): **{len(a['dead_rows'])}**  [GATED - report only]")
+    L.append(f"- Orphaned rows (source_type=orphaned): **{len(a['orphaned_rows'])}**  [GATED - report only]")
+    L.append(f"- Image rows missing AI captions: **{len(a['missing_caption_rows'])}**  [caption these]")
+    L.append("")
+
+    def block(title, items, fmt, limit=40):
+        L.append(f"## {title} ({len(items)})")
+        if not items:
+            L.append("_none_"); L.append(""); return
```

Ruff (0.15.15)

[error] Multiple statements on one line (semicolon)

(E702)